client/components/ReviewRating/ReviewList/Review.jsx

```diff
@@ -11,7 +11,7 @@ import StarRating from '../../StarRating';
 const Review = ({ review, fetchReviews }) => (
   <div className="review">
     <div className="rating_reviewer_date">
-      <StarRating size={10} rating={review.rating} />
+      <StarRating size={16} rating={review.rating} />
```

Comment:
Can we create a constant variable for this size? `const STAR_SIZE = 16;`. I think it would improve readability for future use.

Reply:
sure, keep it on best practice.